behind-the-veil-siteroot/imports/ui/components/pages/profile/settings/ProfileSettingsPage.jsx

```diff
@@ -9,25 +9,24 @@ import PageLayout from "../../../../enums/PageLayout";
 import Tabs from "../../../tabs/Tabs.jsx";
 import AccountDetails from "./AccountDetails.jsx";
 import ArtistServiceArea from "./ArtistServiceArea";
-import {useUserInfo} from "../../../util";
 import ChangePasswordTab from "./ChangePasswordTab";
 import BackButton from "../../../button/BackButton";
 import ArtistPayment from "./ArtistPayment.jsx";
 
-
+import { useTracker } from 'meteor/react-meteor-data';
 /**
  * Settings page for all users
  */
 export const ProfileSettingsPage = () => {
-
+    
     // get current user information
-    const userInfo = useUserInfo();
+    const userInfo = useTracker(() => Meteor.user());
```

Comment:
Just wondering what the reason is for this change. Did the useUserInfo hook not work?

Reply:
yeah the useUserInfo hook doesn't return the service area and payment

Reply:
Have you tried exposing service area and payment from the useUserInfo hook? Otherwise, in the future, anyone else who wants to get service area or payment will have to also do it this way, instead of just getting all user info from one hook

Reply:
I've tried that now and pushed the new commit